.eslintrc.json

```diff
@@ -53,6 +53,7 @@
         "functional/functional-parameters": "off",
         "functional/no-expression-statement": "off",
         "functional/no-return-void": "off",
+        "functional/no-let": "off",
```

Comment:
this is mostly covered by the `prefer-const` rule that we are following (confirmed this is working in the repo already).  Banishing `let`s completely makes the coding more academic than practical (e.g.we would need to `reduce` the lines of our gitignore file or something crazy like that :))

Reply:
I'd still like to keep the rule as `let` could lead to poorly written and hard to follow code. I'll suggest something below to use instead to save you from re-typing that ;-)

Reply:
Could we keep let disabled then and remove this line? :-)

Reply:
sure.

in part I feel dis-allowing `let`s is extreme. but I also agreee that code with `const`s is sometimes cleaner and more correct (albeit sometimes littered with `reduce`s and splatting).